Aptoide/aptoide-client-v8
# Feature/an 1638 show feedback on comment on non social cards

What does this PR do?

   This pull-request changes the feedback when doing a comment on non-social posts. After submitting comment, it will show up visible on the post itself as a form of feedback.

   Some adjustments to the model had to be made to enable the possibility to do this.

Where should the reviewer start?

- [ ] TimelinePresenter : line 674  commentPostResponse()

How should this be manually tested?

  Open Aptoide v8 > Timeline > Find any non-social post (Aptoide recommends/Popular app among friends for example) > comment > wait for feedback (should be the comment on the post).

What are the relevant tickets?

Tickets related to this pull-request: [AN-1638](https://aptoide.atlassian.net/browse/AN-1638)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
